backend/util/tracer-graphql.go

```diff
@@ -73,17 +74,10 @@ func (t Tracer) InterceptResponse(ctx context.Context, next graphql.ResponseHand
 	if oc != nil {
 		opName = oc.OperationName
 	}
-	span, ctx := StartSpanFromContext(ctx, "graphql.operation", ResourceName(opName))
-	span.SetAttribute("backend", t.serverType)
-	defer span.Finish()
+	span, ctx := tracer.StartSpanFromContext(ctx, "graphql.operation", tracer.ResourceName(opName))
+	span.SetTag("backend", t.serverType)
 	resp := next(ctx)
-	if resp != nil {
-		var errs []error
-		for _, err := range resp.Errors {
-			errs = append(errs, err)
-		}
-		span.Finish(errs...)
-	}
```

Comment:
don't record response errors because gqlgen doesn't have a stack trace for them, and they are duplicates of the `field` errors that are reported above in `InterceptField`